py/htm/advanced/examples/thalamus/run_thalamus_experiment.py

```diff
@@ -0,0 +1,261 @@
+# ----------------------------------------------------------------------
+# HTM Community Edition of NuPIC
+# Copyright (C) 2019, Numenta, Inc. 
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.    If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+
+
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from PIL import Image
+
+from scipy import ndimage as ndi
```

Comment:
this also needs scipy, matplotlib, PIL -- all deps of scikit-learn, so yes, go ahead and use the pip install, rather than a borrowed file (Gabor f).

Reply:
Sure. With pleasure.